components/gitpod-cli/cmd/ports-expose.go

```diff
@@ -25,22 +25,21 @@ var portExposeCmd = &cobra.Command{
 	Short: "Makes a port available on 0.0.0.0 so that it can be exposed to the internet",
 	Long:  ``,
 	Args:  cobra.RangeArgs(1, 2),
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// ignore trace
+		utils.TrackCommandUsageEvent.Command = nil
```

Comment:
@iQQBot is it on purpose here? It seems to be a user called? Let's remove?

https://www.gitpod.io/docs/references/gitpod-cli#expose